### PR TITLE
Onetechnical/relstable2.1.6

### PR DESCRIPTION
**Pull request by onetechnical**
## Highlights

A bug in the algorand debian package would result in the genesis.json for TestNet being applied instead of MainNet. Debian users who upgraded to 2.1.5 should verify that `/var/lib/algorand/genesis.json` is pointing to the correct network after upgrading to this version.

## Changes

1. Other
    * Bug Fixes
        * Use DEFAULT_RELEASE_NETWORK genesis file in debian package
        * Attempt to fix bad genesis file on 2.1.5 debian install

## Test Plan

Standard release testing.

clean install 2.1.4, upgrade 2.1.5, upgrade 2.1.6 (scenario #1)
* verify mainnet config (after 2.1.4)
* verify testnet config (after 2.1.5)

clean install 2.1.4, testnet config, upgrade 2.1.5, upgrade 2.1.6 (scenario #2)
* verify mainnet config (after 2.1.4)
* implement testnet config
* verify testnet config (after 2.1.5)
* verify testnet config (after 2.1.6)

clean install 2.1.5, mainnet config, upgrade 2.1.6 (scenario #3)
* verify testnet config (after 2.1.5)
* implement mainnet config
* verify mainnet config (after 2.1.6)

clean install 2.1.5, upgrade 2.1.6 (scenario #4)
* verify testnet config (after 2.1.5)
* verify testnet config (after 2.1.6)

clean install 2.1.6
* verify mainnet config

clean install 2.1.4, upgrade 2.1.6
* verify mainnet config (after 2.1.4)
* verify mainnet config (after 2.1.6)

clean install 2.1.4, testnet config, upgrade 2.1.6
* verify mainnet config (after 2.1.4)
* implement testnet config
* verify testnet config (after 2.1.6)

### CODE REVIEW

**Comment by onetechnical** [comment]
Waiting for betanet validation before changing from draft mode.

File: installer/debian/algorand/preinst [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
 
-        if [ -n "$INSTALLED_PKG" ]
+        if [ "$ALGO_MD5" = "9afc34b96a2c4a9f936870cd26ae7873" ]
         then
-            echo -e "\nAlgorand does not currently support multi-distribution installations!\n\
-    To install this package, it is necessary to first remove the \`$INSTALLED_PKG\` package:\n\n\
-    sudo apt-get remove $INSTALLED_PKG\n\
-    sudo apt-get install $PKG_NAME\n"
-            exit 1
+            if [[ "$ALGO_TIMESTAMP" != 1600456830 ||
```
Comment (bricerisingalgorand) [inline installer/debian/algorand/preinst:39]: is this stuff checking that the last time the genesis file was edited is the right date?

**Review by bricerisingalgorand — APPROVED** [review]

**Comment by onetechnical** [comment]
BetaNet updated successfully. Tested running a BetaNet node using debian package, RPM package, and ARM binary. Considering this safe enough to move to build phase.

File: installer/debian/algorand/preinst (hunk 1 above)
Comment (btoll) [reply installer/debian/algorand/preinst:39]: That timestamp is the timestamp of the 2.1.5 genesis files only, and we're using to check if the install was prior to or after 2.1.5.

**Comment by btoll** [comment]
Tested all scenarios with a 2.1.6 build on both 18.04 and 20.04 and they worked as expected.

**Comment by onetechnical** [comment]
Re-verified all test cases on ubuntu 20.04 on EC2.

**Pull request merged**